p4rt_app/p4runtime/packetio_helpers.h

```diff
@@ -25,6 +25,8 @@
 
 namespace p4rt_app {
 
+#define SEND_TO_INGRESS_PORT_NAME "Ethernet"
```

Comment:
Pls change this to SEND_TO_INGRESS.

Reply:
Actually I meant changing the value of the macro from Ethernet to SEND_TO_INGRESS. I am merging this PR but pls fix it in the next one.
#define SEND_TO_INGRESS_PORT_NAME "SEND_TO_INGRESS"

Reply:
@kishanps, I have changed the macro from "SEND_TO_INGRESS" to "SEND_TO_INGRESS_PORT_NAME" and replaced "Ethernet" with "SEND_TO_INGRESS".